src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -34,6 +34,10 @@ public class HttpSysOptions
     /// </summary>
     public HttpSysOptions()
     {
+        // this feature is being back-ported to net6/net7 via an app-context switch; respect that even on net8+ to avoid
+        // surprises when upgrading TFMs
+        const string EnableKernelResponseBufferingSwitch = "Microsoft.AspNetCore.Server.HttpSys.EnableKernelResponseBuffering";
+        EnableKernelResponseBuffering = AppContext.TryGetSwitch(EnableKernelResponseBufferingSwitch, out var enabled) && enabled;
```

Comment:
It's standard practice to remove app context switches in the next major release (8.0). They're only used for servicing.